max-wave.lua

```diff
@@ -0,0 +1,63 @@
+--Dynamically limit the next immigration wave
+--By Loci, modified by Fleeting Frames and Tachytaenius
+--[====[
+
+max-wave
+========
+Limit the number of migrants that can arrive in the next wave by
+overriding the population cap value in data/init/d_init.txt.
+Not safe with gui/settings-manager as it will commit temporary
+population cap changes permanently to file.
```

Comment:
How do we get from disk though? Lua file IO?

Reply:
Yeah, it's reading the file anyway when it updates the settings. It can pre-read to find discrepancies. This can actually work for all settings, not just population cap

Reply:
after discussing on discord, I think we're wrong here about the danger. `gui/settings-manager` will display the value in the `d_init.txt` file. it will overwrite the pop cap that this script sets, not the other way around. We can just add to this tool's documentation that if you edit the population cap with `gui/settings-manager` after running max-wave, your population cap will be reset and you may get more migrants than you expect. We should similarly add a line to `gui/settings-manager`'s docs (in this PR) that editing the population cap will override any modifications made by scripts such as `max-wave`.